.eslintignore

```diff
@@ -0,0 +1,4 @@
+public/assets/javascripts/webtrends/**
```

Comment:
I would actually be tempted to put this in a `vendor` subfolder now seeing this because we can ignore everything from 3rd parties then and not have to change this file each time.